webapp/tests/near-nep21-util.test.js

```diff
@@ -30,4 +30,21 @@ describe("Normalize Accounts ", () => {
         var res = normalizeAmount("42");
         assert.equal(res, "42000000000000000000000000", 'mismatch');
     });
+});
+
+describe("Convert to real number", () => {
+
+    it("Division ",  () => {
+        var res = convertToReal("1000000000000000000000000");
+        assert.equal(res, "1", 'mismatch');
+        
+        var res = convertToReal("10000000000000000000000");
+        assert.equal(res, "0.01", 'mismatch');
+
+        var res = convertToReal("1234567891234567891234567");
+        assert.equal(res, "1.234567891234567891234567", 'mismatch');
+
+        var res = convertToReal("1234567891234567894567");
+        assert.equal(res, "0.001234567891234567894567", 'mismatch');
```

Comment:
please add a test for the following values:

+ `0`
+ `1`
+ `00`
+ `0012340000000000000000000000000`
+ `0012340000000000000000000000001`

Reply:
0 and 00 ,I think it is checked that we pass only positive values in UI text box, 
but  i will add these cases too for safety